lib/src/blocs/auth_bloc/auth_bloc.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
```

Comment:
why we have auth_bloc.dart file in auth_bloc folder? is it make sense?

Reply:
auth_bloc folder was deleted because no longer used